### ISSUE TITLE
Paragraph in *conversation* is wider but its container if containing big images

### ISSUE THREAD

**Issue by DeusFigendi** [diaspora/diaspora#6823]
Okay first of all I'm talking about Code 0.5.99.0-pabca848b running on Geraspora at the moment.

It seems when a conversation message contains an image and that image has a higher width but... 650px (or the like) it does not shrink to fit into the *stream_element message* it just overflows hidden.

Bad enough it also affects paragraphs to do the same.

[ ![Demo](https://pod.geraspora.de/uploads/images/scaled_full_f549b88cfa880f51a8c8.png) ](https://pod.geraspora.de/uploads/images/f549b88cfa880f51a8c8.png) (click image to open full size)

and this is the markdown I used:

````
1 Officia reiciendis quisquam consequatur sed nulla eos culpa asperiores. 2 Aut sit veniam adipisci. 3 Voluptatem et ut ea. 4 Nihil facere ut porro. 5 Cupiditate at aut non molestiae. 6 Magnam pariatur voluptatum sed. 7 Dolor dolores porro voluptate dignissimos. 8 Tempora est repellendus rerum. 9 Earum facere iste inventore temporibus. 10 Ad eius expedita odit. 11 Tenetur voluptatem inventore repellat. 12 Repellendus reprehenderit quo est. 13 Suscipit dolorum itaque magni omnis qui error rerum libero. 14 Quae quis aut amet. 15 Aut ab dolor aut ut alias. 16 Blanditiis id omnis animi numquam a dolor id. 17 Odio consequuntur pariatur a. 18 Aliquid ullam sit vero asperiores et cupiditate ut voluptates. 19 Aut qui ipsum a culpa quia quibusdam sit harum. 20 Dolor quidem provident sit exercitationem sunt voluptatibus aut.
````

and

````
1 Officia reiciendis quisquam consequatur sed nulla eos culpa asperiores. 2 Aut sit veniam adipisci. 3 Voluptatem et ut ea. 4 Nihil facere ut porro. 5 Cupiditate at aut non molestiae. 6 Magnam pariatur voluptatum sed. 7 Dolor dolores porro voluptate dignissimos. 8 Tempora est repellendus rerum. 9 Earum facere iste inventore temporibus. 10 Ad eius expedita odit.

![1000 x 1000](https://upload.wikimedia.org/wikipedia/commons/5/53/Lorem_Ipsum_Gill_Sans.png)

11 Tenetur voluptatem inventore repellat. 12 Repellendus reprehenderit quo est. 13 Suscipit dolorum itaque magni omnis qui error rerum libero. 14 Quae quis aut amet. 15 Aut ab dolor aut ut alias. 16 Blanditiis id omnis animi numquam a dolor id. 17 Odio consequuntur pariatur a. 18 Aliquid ullam sit vero asperiores et cupiditate ut voluptates. 19 Aut qui ipsum a culpa quia quibusdam sit harum. 20 Dolor quidem provident sit exercitationem sunt voluptatibus aut.
````

As you can see the image is displayed too wide and the first paragraph does not linebreak at `3 Voluptatem et ut ea.`, instead the 4^th and 5^th sentence is missing. The image is 1000px x 1000px.

**Comment by denschub** [comment]
Also, this affects code blocks. Thanks for the report!

**Comment by denschub** [comment]
Closed via #6831.
